beacon-chain/p2p/rpc_topic_mappings.go

```diff
@@ -290,13 +306,25 @@ func (r RPCTopic) Version() string {
 // TopicFromMessage constructs the rpc topic from the provided message
 // type and epoch.
 func TopicFromMessage(msg string, epoch primitives.Epoch) (string, error) {
+	// Check if the topic is known.
 	if !messageMapping[msg] {
 		return "", errors.Errorf("%s: %s", invalidRPCMessageType, msg)
 	}
+
+	// Base version is version 1.
 	version := SchemaVersionV1
+
+	// Check if the message is to be updated in altair.
 	isAltair := epoch >= params.BeaconConfig().AltairForkEpoch
 	if isAltair && altairMapping[msg] {
 		version = SchemaVersionV2
 	}
+
+	// Check if the message is to be updated in fulu.
+	isFulu := epoch >= params.BeaconConfig().FuluForkEpoch
+	if isFulu && fuluMapping[msg] {
+		version = SchemaVersionV3
```

Comment:
This line is untested. Is it important?

![Screenshot_2025-05-27_16-13-59](https://github.com/user-attachments/assets/fabbaecc-b729-4263-9f9d-53bc432fcc51)